src/ui/public/styles/theme.less

```diff
@@ -174,25 +175,16 @@ table,
       color: @table-special-color;
     }
   }
+}
 
-  // > thead > tr > th,
-  // > tbody > tr > th,
-  // > tfoot > tr > th,
-  // > thead > tr > td,
-  // > tbody > tr > td,
-  // > tfoot > tr > td {
```

Comment:
I totally agree about the pain here, but I hesitate to dig into it without understanding the full scope of these styles. I'd prefer to handle this in its own branch.